cmd/demo/node.go

```diff
@@ -445,15 +459,24 @@ func (n *node) Info(args []string) error {
 	defer n.mtx.Unlock()
 	n.log.Traceln("Info...")
 
+	ctx, cancel := context.WithTimeout(context.Background(), config.Chain.DeployTimeout)
+	defer cancel()
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', tabwriter.Debug)
-	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\t\n")
+	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\tMy On-Chain Ξ\tPeer On-Chain Ξ\t\n")
 	for alias, peer := range n.peers {
+		onChainBals, err := getOnChainBal(ctx, n.onChain.Address(), peer.perunID)
+		if err != nil {
+			return err
+		}
+
 		if peer.ch == nil {
-			fmt.Fprintf(w, "%s\t%s\t\n", alias, "Connected")
+			fmt.Fprintf(w, "%s\t%s\t \t \t \t%v\t%v\t\n", alias, "Connected",
+				weiToEther(onChainBals[0])[0], weiToEther(onChainBals[1])[0])
 		} else {
 			bals := weiToEther(peer.ch.GetBalances())
-			fmt.Fprintf(w, "%s\t%v\t%d\t%v\t%v\t\n",
-				alias, peer.ch.Phase(), peer.ch.State().Version, bals[0], bals[1])
+			fmt.Fprintf(w, "%s\t%v\t%d\t%v\t%v\t%v\t%v\t\n",
+				alias, peer.ch.Phase(), peer.ch.State().Version, bals[0], bals[1],
+				weiToEther(onChainBals[0])[0], weiToEther(onChainBals[1])[0])
```

Comment:
same as above, use variadic converter properly
```suggestion
				weiToEther(onChainBals)...)
```

Reply:
Done, see above.